src/commands/small.rs

```diff
@@ -117,3 +117,10 @@ pub async fn dotfiles(ctx: &client::Context, msg: &Message, args: Args) -> Comma
 
     Ok(())
 }
+
```

Comment:
Please add a doc comment describing what it does, which will also be shown in the !help output